README

```diff
@@ -35,14 +58,72 @@ Connect client:
 $ ./src/tools/perf/ucx_perftest <server-hostname> -t tag_lat -c 0
 ```
 
-### UCX high-level layout
-UCX contains four main components:
-- UCP - UCX Protocol. Implements high-level abstractions such as tag-matching,
-  streams, connection negotiation and establishment, multi-rail, and handling 
-  different memory types.
-- UCT - UCX Transport. Implements low-level communication primitives such as
-  active messages, remote memory access, and atomic operations.
-- UCS - UCX Services. A collection of data structures, algorithms, and system
-  utilities for common use.
-- UCM - UCX Memory. Intercepts memory allocation and release events, used by
-  memory registration cache.
+## Our Community
+
+* [Project Website](http://www.openucx.org/)
+* [Github](http://www.github.com/openucx/ucx/)
+* [Software Releases](http://www.github.com/openucx/ucx/releases)
+* [Mailing List](https://elist.ornl.gov/mailman/listinfo/ucx-group)
+
+## Licenses
+
+UCX comes with 2 licenses:
+
+* [BSD3](LICENSE)
+* [Apache 2.0](LICENSES.apache20)
```

Comment:
Our license is BSD3 only. I think we have only one file is re-distributed under Apache.
@yosefe I think we have explicitly list that only one particular file is re-distributed under apache.

Reply:
there is PR to remove Apache file from the repo. So, let't go ahead and remove the reference to Apache License.